components/admin_console/s3_connection_test/index.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+import {testS3Connection} from 'mattermost-redux/actions/admin';
+
+import S3ConnectionTest from './s3_connection_test.jsx';
+
+function mapStateToProps(state, ownProps) {
+    return {
+        ...ownProps
+    };
+}
```

Comment:
One last small request, this function actually isn't need this case, so we can remove it and just do at the bottom `export default connect(null, mapDispatchToProps)(S3ConnectionTest);`

Reply:
ok, I will update, thanks!

Reply:
and @jwilander shall we make the same changes to the test email?

Reply:
Yeah if you want go for it :)

Reply:
will do that in the weekend

Reply:
done the request above.